code/modules/reagents/chemistry_properties/prop_neutral.dm

```diff
@@ -519,6 +519,31 @@
 /datum/chem_property/neutral/hyperthrottling/process_critical(mob/living/M, potency = 1, delta_time)
 	M.apply_effect(potency * delta_time, PARALYZE)
 
+/datum/chem_property/neutral/encephalophrasive
+	name = PROPERTY_ENCEPHALOPHRASIVE
+	code = "ESP"
+	description = "Drastically increases the amplitude of Gamma and Beta brain waves, allowing the host to broadcast their mind."
+	rarity = PROPERTY_LEGENDARY
+	category = PROPERTY_TYPE_STIMULANT
+	value = 8
+
+/datum/chem_property/neutral/encephalophrasive/on_delete(mob/living/M)
```

Comment:
make your Ms something better, like affected_mob

Reply:
I handled it the exact same as all other chemical properties. I feel it would be odd if it were changed specifically for encephalophrasive an not on anything else, for example

`/datum/chem_property/positive/anticorrosive/process(mob/living/M, potency = 1)`

Reply:
leaving single letter vars because the whole file is single letter is not an excuse to not change them to something good!!

Reply:
'M' changed to 'chem_host'